src/pages/RegisterPage.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div class="register-page">
+    <h1 class="register-page__ahoy">Ahoy you!</h1>
+    <h1 class="register-page__care">Care to register?</h1>
```

Comment:
it is a good practice to have only one `<h1>` tag on a page, maybe there is some way to achieve it 🤔